### DISCUSSION TITLE
How can I prevent certain elements from fading in Azutacast?

### DISCUSSION THREAD

**Discussion by RadioHerzblut** [AzuraCast/AzuraCast#6752] [General]
How can I prevent certain elements from fading in Azutacast?
Sometimes it works, sometimes not really

![image](https://github.com/AzuraCast/AzuraCast/assets/150465082/9c8f0055-dab6-4f1f-a5d2-9b22f8ed1876)

These settings work well from time to time.
Am I doing something wrong?

**Comment by LordHelmchen666** [comment]
Moonbase59 suggested somewhere to Set Einblendezeit, Ausblendezeit and Überschneidungszeit to 0.1 because LS sometimes does not respect 0 as value.
I set my files with 0.1 but i also realize the behaviour you described. Sometimes it works sometimes not. But this is an Liquidsoap issue.

**Reply by RadioHerzblut** [reply]
Currently, the "non-blending" of the special elements works with the parameter you suggested. 
Let's hope it stays that way ;-)

**Reply by RadioHerzblut** [reply]
Next Update:  Work PERFEKT - Thanks Lord Helmchen
